pkg/common/oras/authprovider/aws/awsecrbasic.go

```diff
@@ -92,13 +94,24 @@ func getEcrAuthToken() (EcrAuthToken, error) {
 		return EcrAuthToken{}, fmt.Errorf("failed to load default AWS basic auth config: %w", err)
 	}
 
+	// registry/region from image
+	registry, err := awsauth.RegistryFromImage(artifact)
```

Comment:
nit: I believe convention is usually to error handle immediately after the line that produces the `err` variable